lib/rules/no-unexpected-multiline.js

```diff
@@ -24,11 +24,13 @@ module.exports = function(context) {
      */
     function checkForBreakAfter(node, msg) {
         var paren = context.getTokenAfter(node);
+        var beforeParen = node;
         while (paren.value === ")") {
+            beforeParen = paren;
             paren = context.getTokenAfter(paren);
```

Comment:
For future reference, I was not right. The `paren` variable and the comparison to `)` confused me.